frontend/javascripts/test/sagas/save_mutex_saga.spec.ts

```diff
@@ -119,13 +126,38 @@ describe("Annotation Saga", () => {
   });
 
   it<WebknossosTestContext>("An annotation with allowUpdate = true and othersMayEdit = false should not try to acquire the annotation mutex.", async (context: WebknossosTestContext) => {
+    WkDevFlags.liveCollab = false;
     // allowUpdate = true and othersMayEdit = false are the defaults for the annotation,
     // no manual changes via a function passed to setupWebknossosForTesting is needed
     await setupWebknossosForTesting(context, "hybrid");
     expect(context.mocks.acquireAnnotationMutex).not.toHaveBeenCalled();
   });
 
-  it<WebknossosTestContext>("An annotation with isUpdatingCurrentlyAllowed = true and othersMayEdit = true should not try to acquire the annotation mutex.", async (context: WebknossosTestContext) => {
+  it<WebknossosTestContext>("An annotation with isUpdatingCurrentlyAllowed = true and othersMayEdit = true should try to acquire the annotation mutex in liveCollab scenario.", async (context: WebknossosTestContext) => {
+    WkDevFlags.liveCollab = true;
+    await setupWebknossosForTesting(
+      context,
+      "hybrid",
+      ({ tracings, annotationProto, dataset, annotation }) => {
+        const annotationWithUpdatingAllowedTrue = update(annotation, {
+          restrictions: { allowUpdate: { $set: true }, allowSave: { $set: true } },
+          othersMayEdit: { $set: true },
+        });
+        return {
+          tracings,
+          annotationProto,
+          dataset,
+          annotation: annotationWithUpdatingAllowedTrue,
+        };
+      },
+    );
+    expect(context.mocks.acquireAnnotationMutex).toHaveBeenCalled();
+    const isUpdatingAllowed = Store.getState().annotation.isUpdatingCurrentlyAllowed;
+    expect(isUpdatingAllowed).toBe(true);
+  });
+
+  it<WebknossosTestContext>("An annotation with isUpdatingCurrentlyAllowed = true and othersMayEdit = true should try to acquire the annotation mutex not in liveCollab.", async (context: WebknossosTestContext) => {
```

Comment:
"not in liveCollab" is very hard to read. how about:
A not-liveCollab annotation with isUpdatingCurrentlyAllowed = true and othersMayEdit = true should try to acquire the annotation mutex.